ocs_ci/ocs/constants.py

```diff
@@ -945,3 +945,25 @@
 
 # cluster expansion
 MAX_OSDS = 15
+
+# Bare Metal constants
+bm_httpd_server = "10.8.128.12"
+bm_path_to_upload = "/var/www/html/pratik/ocs4qe/"
+bm_httpd_server_user = "root"
+bm_tftp_base_dir = "/var/lib/tftpboot/"
+bm_tftp_dir = bm_tftp_base_dir + "ocs4qe/baremetal"
+bm_dnsmasq_dir = "/etc/dnsmasq.d/"
```

Comment:
I am not sure it is good to save these details here. Can we maybe save these as conf files that are available only internally? Similar to how we do with credential files for vsphere

+ @red-hat-storage/ocs-qe-ecosystem

Reply:
@prsurve sensitive info I see is ip address of bm_httpd_server, may be you can consolidate all sensitive info for bm config and put it in auth.yaml on jenkins and also in google drive.

Reply:
Done